planemo/options.py

```diff
@@ -1607,7 +1613,6 @@ def test_options():
             help="Update test-data directory with job outputs (normally"
             " written to directory --job_output_files if specified.)",
         ),
-        paste_test_data_paths_option(),
```

Comment:
Not needed any more?

Reply:
It is, I didn't mean to remove that from all test like commands